scripts/healthcheck.sh

```diff
@@ -59,7 +59,7 @@ if [[ ${WEBPROXY_ENABLED} =~ [yY][eE]?[Ss]?|[tT][Rr][Uu][eE] ]]; then
   if [[ ${PROXY} -eq 0 ]]; then
     echo "Privoxy warning: process was stopped, restarting."
   fi
-    proxy_ip=$(grep -oP "(?<=^listen-address).*$" /etc/privoxy/config | sed 's/ //g')
+    proxy_ip=$(grep -oP "(?<=^listen-address).*$" /etc/privoxy/config | sed 's/\:.*//' | sed 's/ //g')
```

Comment:
would a grep like this one be easier to read ? 
 `grep -oP "(?<=^listen-address )[0-9\.]+" /etc/privoxy/config`